packages/web-component/src/lib/helpers/helpers.ts

```diff
@@ -360,3 +416,25 @@ export const getInputValueByType = (input: HTMLInputElement): Promise<any> =>
       }
     }
   });
+
+export const injectSamlIdpForm = (samlIdpFormResponse: string) => {
+  // create form tag from response string
+  const htmlFormTemp = document.createElement('div');
+  htmlFormTemp.innerHTML = samlIdpFormResponse;
+  const htmlForm = htmlFormTemp.querySelector('form');
+  document.body.appendChild(htmlForm);
+
+  // create script tags from response scripts
+  const htmlScripts = document.createElement('div');
+  htmlScripts.innerHTML = samlIdpFormResponse;
+  const scripts = Array.from(htmlScripts.getElementsByTagName('script'));
+  scripts.forEach((originalScript) => {
+    const script = document.createElement('script');
+    if (originalScript.src) {
+      script.src = originalScript.src;
+    } else {
+      script.textContent = originalScript.textContent;
```

Comment:
We might have issues with the [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) here 
@omercnet WDYT?

Reply:
what's going on here?

Reply:
We are generating a script tag and setting its content to a string we got from the server

Reply:
During runtime in the client?! Very sus. 
Can't the script itself conditionally load the right code ?

Reply:
finally we are generating the form inside the Webcomponent (and getting the url, samlResponse, relayState from orchestration service).